request_a_govuk_domain/request/models/organisation.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+
+class Organisation(models.Model):
+    name = models.CharField()
+
+
+class RegistrantTypeChoices(models.TextChoices):
+    central_goverment = "Central government department or agency"
+    ndpb = "Non-departmental body - also known as an arm's length body"
+    fire_service = "Fire Service"
+    local_authority = "County, borough, metropolitan or district council"
+    parish_council = "Parish, town or community council"
+    village_council = "Neighbourhood or village council"
+    combined_authority = "Combined or unitary authority"
+    pcc = "Police and Crime Commissioner"
+    joint_authority = "Joint Authority"
+    joint_committee = "Joint Committee"
+    representative = "Representing public sector bodies"
+    other = "None of the above"
```

Comment:
We have a different list here: https://github.com/co-cddo/domains-register-a-govuk-domain/blob/main/request_a_govuk_domain/request/forms.py#L200

which I'll correct as the list there was based on the existing list in the front-end.

A minor point is that per Mural, "None of the Above" will take us to error page https://app.asana.com/0/1206751381032462/1206751830401990/f

so will we have 'other' as a value ever?